ms_game_chat/backend/backend/urls.py

```diff
@@ -19,7 +19,7 @@
 
 
 urlpatterns = [
-    path('admin/', admin.site.urls),
+    path('game/admin', admin.site.urls),
```

Comment:
@kris-bo @mariekart42 now you can access the admin page through the proxy and dont need to open ports to the host system anymore `/game/admin`